packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -368,6 +370,42 @@ export default class BackupNg {
     return job
   }
 
+  async checkAuthorizations({ job, useSmartBackup, schedule }) {
+    const { _app: app } = this
+
+    if (job.mode === 'full') {
+      await app.checkFeatureAuthorization('BACKUP.FULL')
+    }
+
+    if (job.mode === 'delta') {
+      if (unboxIdsFromPattern(job.srs)?.length > 0) {
+        await app.checkFeatureAuthorization('BACKUP.DELTA_REPLICATION')
+      } else {
+        await app.checkFeatureAuthorization('BACKUP.DELTA')
+      }
+    }
+
+    if (job.type === 'metadataBackup') {
+      await app.checkFeatureAuthorization('BACKUP.METADATA')
+    }
```

Comment:
Either `return` here or put the rest of the code in a `else` clause as it does not apply to metadata backups.